elephant/spade.py

```diff
@@ -1001,18 +1062,30 @@ def approximate_stability(concepts, rel_matrix, n_subsets, delta=0, epsilon=0):
 
     Parameters
     ----------
-    concepts:
-
+    concepts: ???
+    rel_matrix: ???
     n_subsets: int
-        Number of iterations to find an approximated stability.
+        Number of subsets of a concept used to approximate its stability. If
+        n_subset is set to 0 the stability is not computed. If, however,
+        for parameters delta and epsilon (see below) delta + epsilon == 0,
+        then an optimal n_subsets is calculated according to the formula given
+        in Babin, Kuznetsov (2012), proposition 6:
+
+         ..math::
+                n_subset = frac{1}{2\eps^2} \ln(frac{2}{\delta}) +1
+
+        Default:0
     delta: float
-        Probability with at least ..math:$1-\delta$
+        delta: probability with at least ..math:$1-\delta$
+        Default: 0
     epsilon: float
-        Absolute error
-    If delta + epsilon == 0 then an optimal n_subset is calculated according to
-    the formula given in Babin, Kuznetsov (2012) (Proposition 6):
-     ..math::
-                n_subset = frac{1}{2\eps^2} \ln(frac{2}{\delta}) +1
+        epsilon: absolute error
+        Default: 0
+
+    Output
+    ------
+    stability: list
```

Comment:
Can you please specify the output a bit more in detail?

Reply:
Addressed in the following commit (858c363). Thanks for comments!